src/test/java/org/mule/tools/devkit/sonar/checks/maven/DevkitLatestVersionCheckTest.java

```diff
@@ -0,0 +1,57 @@
+package org.mule.tools.devkit.sonar.checks.maven;
+
+
+import com.google.common.collect.Iterables;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.junit.Test;
+import org.mule.tools.devkit.sonar.checks.ConnectorIssue;
+import org.mule.tools.devkit.sonar.utils.PomUtils;
+
+import javax.xml.stream.XMLStreamException;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.hamcrest.Matchers.emptyIterable;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class DevkitLatestVersionCheckTest {
+
+    @Test
+    public void checkDevkitVersionIsLatest() throws IOException, XmlPullParserException, XMLStreamException {
+        final MavenProject mavenProject = PomUtils.createMavenProjectFromPomFile(new File("src/test/files/maven/devkit-latest-version/devkit-version-is-latest"));
```

Comment:
You can separate the first three lines of this method as something like:
```java
private Iterable<ConnectorIssue> analyze(String fileName)
```
and reuse it in all three test cases.